management/grant.go

```diff
@@ -29,7 +29,7 @@ func newGrantManager(m *Management) *GrantManager {
 //
 // See: https://auth0.com/docs/api/management/v2#!/Grants/get_grants
 func (m *GrantManager) List(opts ...RequestOption) (g []*Grant, err error) {
-	err = m.Request("GET", m.URI("grants"), &g, opts...)
+	err = m.Request("GET", m.URI("grants"), &g, applyListDefaults(opts))
```

Comment:
`applyListDefaults` will add the `include_totals=true` query parameter to the request. It's needed for pagination. The response will differ in that case from an array to an object containing -among other fields- a `grands` array.

```json
{
  "total": 0,
  "start": 0,
  "limit": 50,
  "grants": []
}
```

This means we'll need to create a `GrantList` struct to return here instead of `[]*Grant`.

```go
type GrantList struct {
	List
	Grants []*Grant `json:"grants"`
}
```

Reply:
Ah yes! That makes sense. Added.